js/gated-content/src/scss/_fonts.scss

```diff
@@ -0,0 +1,2 @@
+$cachet-book: "Cachet Book", "Cachet W01 Book", "Cachet", Verdana, sans-serif;
+$cachet-medium: "Cachet Medium", "Cachet W01 Medium","Cachet" Verdana, sans-serif;
```

Comment:
Haven't you missed the comma between "Cachet" and Verdana?

Reply:
Yes, thanks! Fixed